### PR TITLE
[unit-tests-only] Added unit tests todo items for resources helper

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/tests/unit/helpers/resources.spec.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  })
+
+  describe('attachIndicators', () => {
+    it.todo(
+      'should set resource "indicator" as indicators returned from "getIndicators" helper method'
+    )
+  })
+
+  describe('aggregateResourceShares', () => {
+    // it.todo('')
```
Comment (pascalwengerter) [inline packages/web-app-files/tests/unit/helpers/resources.spec.js:136]: @kiranparajuli589 what's the idea here? :)

**Review by pascalwengerter — COMMENTED** [review]
Not sure what the ideal way here is. The helper is huge and only partly covered in the outlined test todo-s, but also we're thinking about refactoring this part soon. @kulmann @fschade @paulcod3 opinions?

File: packages/web-app-files/tests/unit/helpers/resources.spec.js (hunk 1 above)
Comment (pascalwengerter) [reply packages/web-app-files/tests/unit/helpers/resources.spec.js:136]: But please don't invest further resources here until we as the `web` team knows what we want - see above

Comment (fschade) [reply packages/web-app-files/tests/unit/helpers/resources.spec.js:136]: i would say skip it or test just the basics, the resources helpers is something we have to refactor

Comment (kiranparajuli589) [reply packages/web-app-files/tests/unit/helpers/resources.spec.js:136]: @pascalwengerter @fschade sorry I forget to follow up on this PR, actually, this was a draft, I was still not finished and got involved with other things.

Are the added todos good for further implementations? or should we just close it and test only after the refactorings are done.

If it's not required for now, feel free to close it _(or just ping me ;))_.

**Comment by kiranparajuli589** [comment]
closed as it seems not required further

**Pull request closed without merging**